apps/mobile/src/components/widgets/account/account-header.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { AccountSelectorSheet } from '@/features/account-selector-sheet';
+import { t } from '@lingui/macro';
+
+import { ChevronRightIcon, SheetRef, Text, TouchableOpacity } from '@leather.io/ui/native';
+
+interface AccountHeaderProps {
+  hasAccounts: boolean;
+  sheetRef: React.RefObject<SheetRef>;
+}
+
+function AccountHeaderText() {
+  return <Text variant="heading05">{t`My accounts`}</Text>;
+}
+
+export function AccountHeader({ hasAccounts, sheetRef }: AccountHeaderProps) {
+  if (!hasAccounts) return <AccountHeaderText />;
+  return (
+    <>
+      <TouchableOpacity
+        onPress={() => sheetRef.current?.present()}
+        flexDirection="row"
+        gap="1"
+        alignItems="center"
+      >
+        <AccountHeaderText />
+        <ChevronRightIcon variant="small" />
+      </TouchableOpacity>
+
+      <AccountSelectorSheet sheetRef={sheetRef} />
```

Comment:
Is this a smart component? If so, should it be passed in a children instead?

Reply:
I'm thinking that perhaps I will integrate this into `Widget` as most of them seem to have headers. 

I'll think about how to improve it and refactor to pass it as children to improve when working on https://github.com/leather-io/issues/issues/138